desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -243,7 +243,7 @@ def _make_unfurl_payload(request, url, id_type, doc, doc_type):
           "type": "section",
           "text": {
             "type": "mrkdwn",
-            "text": "\n*<{}|Open {} {} in Hue - SQL Editor>*".format(url, name, doc_type)
+            "text": "\n*<{}|Open {} {} in Hue | Created by: {}>*".format(url, name, doc_type, created_by),
```

Comment:
Could event be 

`dialect` create by ... 

maybe

Reply:
PS: please use {url} etc and not just {} as it gets not easy to read

Reply:
Fixed